Specs/Scene/PointCloud3DTileContentSpec.js

```diff
@@ -436,6 +438,102 @@ defineSuite([
         });
     });
 
+    function countRenderedPixels(rgba) {
+        var pixelCount = rgba.length / 4;
+        var count = 0;
+        for (var i = 0; i < pixelCount; i++) {
+            var index = i * 4;
+            if (rgba[index] !== 0 ||
+                rgba[index + 1] !== 0 ||
+                rgba[index + 2] !== 0 ||
+                rgba[index + 3] !== 255) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    it('attenuates points based on geometric error', function() {
```

Comment:
Should this be one test?  Or should each call to `expect...` be a separate test so it is easier to track down test failures?